TGM/src/main/java/com/minehut/tgm/modules/DeathModule.java

```diff
@@ -18,14 +16,16 @@
      * This is to prevent players from "combat logging"
      * by team swapping to avoid getting a death.
      */
-    @EventHandler
-    public void onTeamChange(TeamChangeEvent event) {
-        if (TGM.get().getMatchManager().getMatch().getMatchStatus() == MatchStatus.MID) {
-            if (event.getOldTeam() != null && !event.getOldTeam().isSpectator()) {
-                event.getPlayerContext().getPlayer().setHealth(0);
-            }
-        }
-    }
+
+    // not need because i prevent people changing teams during the game
+//    @EventHandler
+//    public void onTeamChange(TeamChangeEvent event) {
+//        if (TGM.get().getMatchManager().getMatch().getMatchStatus() == MatchStatus.MID) {
+//            if (event.getOldTeam() != null && !event.getOldTeam().isSpectator()) {
+//                event.getPlayerContext().getPlayer().setHealth(0);
+//            }
+//        }
+//    }
```

Comment:
You disabled team switching globally? Why would you make such a change in a pull request for code of an individual gamemode? I agree that team switching shouldn't even be a thing, but this isn't the place to make that change.